### PR TITLE
feat: contact page

### PR DESCRIPTION
**Pull request by GugaGongadze**
Adds a contact page with a simple form for users to reach out to Shuttle team.

### CODE REVIEW

File: pages/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+								fill-rule='evenodd'
+								d='M5.23 7.21a.75.75 0 011.06.02L10 11.168l3.71-3.938a.75.75 0 111.08 1.04l-4.25 4.5a.75.75 0 01-1.08 0l-4.25-4.5a.75.75 0 01.02-1.06z'
+								clip-rule='evenodd'
+							/>
+						</svg>
+						<select
+							name='target'
+							id='target'
+							onChange={(e) => setTarget(e.target.value as FormTargetOption)}
+							className='h-[56px] w-full appearance-none rounded-xl border border-slate-700 bg-transparent px-[12px] py-[15px] text-xl font-normal text-[#D8D8D8] placeholder:text-[#D8D8D8]'
```
Comment (chesedo) [inline pages/contact.tsx:43]: Not sure why, but on my system the dropdown text is hard to read
image

File: pages/contact.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+						onChange={(e) => setBody(e.target.value)}
+						name='body'
+						placeholder='Message'
+						className='w-full rounded-xl border border-slate-700 bg-transparent px-[12px] py-[15px] text-xl font-normal text-[#D8D8D8] placeholder:text-[#D8D8D8]'
+						required
+					/>
+					<button
+						type='submit'
+						onClick={handleSubmit}
+						className='mt-4 h-[56px] w-full rounded-[14px] bg-[#D8D8D8] text-lg text-black sm:mt-0'
```
Comment (chesedo) [inline pages/contact.tsx:82]: Can we give the button the same gradient border or orange gradient as the other buttons?

File: pages/contact.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+						placeholder='Subject'
+						className='h-[56px] w-full rounded-xl border border-slate-700 bg-transparent px-[12px] py-[15px] text-xl font-normal text-[#D8D8D8] placeholder:text-[#D8D8D8]'
+						required
+					/>
+					<textarea
+						value={body}
+						onChange={(e) => setBody(e.target.value)}
+						name='body'
+						placeholder='Message'
+						className='w-full rounded-xl border border-slate-700 bg-transparent px-[12px] py-[15px] text-xl font-normal text-[#D8D8D8] placeholder:text-[#D8D8D8]'
```
Comment (chesedo) [inline pages/contact.tsx:76]: The slight blueish borders on the inputs feel a bit out of place. Can we go with a pure white or orangy one instead?

**Review by chesedo — COMMENTED** [review]
Like how simple it is. Left some color comments

File: pages/contact.tsx (hunk 3 above)
Comment (GugaGongadze) [reply pages/contact.tsx]: Remove outline altogether, does not look good no matter the color.

File: pages/contact.tsx (hunk 1 above)
Comment (GugaGongadze) [reply pages/contact.tsx]: What is your OS/browser combo? Looks fine on MacOS>

Comment (jonaro00) [reply pages/contact.tsx]: In my experience, these are typically caused by darkmode plugins

**Comment by oddgrd** [comment]
Hey @GugaGongadze, would you mind linking to the preview page for this?

**Comment by GugaGongadze** [comment]
@oddgrd https://shuttle-main-gugagongadze-getsynth.vercel.app/contact

**Review by oddgrd — APPROVED** [review]
LGTM! Just curious, why are we using the default email client from a users system, rather than using a library to send an email from the form submission directly? Just to make this simpler (I suppose it won't see that much use)?

**Pull request merged**